### PR TITLE
SO-3434: Remove formatting of exported query expression

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java [hunk 1]
```diff
@@ -257,7 +257,15 @@ protected SnomedRefSetMemberSearchRequestBuilder createSearchRequestBuilder() {
 					// Append extra columns using the properties map
 					final Map<String, Object> properties = member.getProperties();
 					for (final String extraColumn : extraColumns) {
-						builder.add(toColumn(properties.get(extraColumn)));
+						Object property = properties.get(extraColumn);
+						if (extraColumn.equals(SnomedRf2Headers.FIELD_QUERY)) {
+							String query = (String) property;
+							query = query.replace("\r", " ");
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java]: A simplified regex would be better here to reduce multiple control characters into a single space: `replaceAll("[\r\n]+", " ")`.

Comment (cmark) [reply snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java]: Also, move the logic to the `toColumn()` method.

**Review by cmark — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java (hunk 1 above)
Comment (apeteri) [inline snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java]: The first replacement could be `query = query.replace("\r\n", " ");`
The way it currently is, CRLF sequences get replaced by two spaces, one for `\r` and one for `\n`.

Comment (apeteri) [reply snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java]: Another option is to use Guava's  `CharMatcher` for all replacements:
```
property = CharMatcher.breakingWhitespace().replaceFrom(query, ' ');
```

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**